pkg/volume/gcepd/attacher.go

```diff
@@ -138,6 +141,52 @@ func (attacher *gcePersistentDiskAttacher) VolumesAreAttached(specs []*volume.Sp
 	return volumesAttachedCheck, nil
 }
 
+
+// search Windows disk number by LUN
+func getDiskID(pdName string, exec mount.Exec) (string, error) {
+	cmd := `Get-GcePdName | select Name, DeviceId | ConvertTo-Json`
+	output, err := exec.Run("powershell", "/c", cmd)
```

Comment:
Add a TODO and a bug number for replacing this code with native support from `Get-Disk`.

Reply:
added